api/model.ts

```diff
@@ -18,6 +32,23 @@ export type CheckpointBlock = {
     timestamp: string
 }
 
+export type Transaction = {
+    hash: string
+    amount: number
+    receiver: string
+    sender: string
+    fee: number
+    isDummy: boolean
+    lastTransactionRef: {
+        prevHash: string
+        ordinal: number
+    }
+    snapshotHash: string
+    checkpointBlock: string
+    transactionOriginal: unknown
```

Comment:
Why `unknown` here?

Reply:
This type doesn't matter because it should be returned "as is"

Reply:
So `any` then. Because `unknown` forces us to cast it to something when accessing